contactus.php

```diff
@@ -42,17 +47,60 @@
         </div>
     </nav>
-    <main>
+    <div id="form" name="form">
+        <?php
+        $fullname = '';
+        $email = '';
+        $subject = '';
+        $message = '';
+        $mailFrom = '';
+
+    if (isset($_Post['submit-btn'])) {
+
+         if (!empty(trim($_POST['name']))) {
+            $fullname = trim($_POST['name']);
+        } else {
+            echo "<p> Name cannot be blank</p>";
+        }
+
+        if (!empty(trim($_POST['subject']))) {
+            $subject = trim($_POST['subject']);
+        } else {
+            echo "<p>Subject cannot be blank</p>";
+        }
+
+        if (!empty(trim($_POST['message']))) {
+            $message = trim($_POST['message']);
+        } else {
+            echo "<p>Message cannot be blank</p>";
+        }
+
+        if (isset($_POST['submit'])) {
+            $fullname = $_POST['name'];
+            $subject = $_POST['subject'];
+            $mailFrom = $_POST['mail'];
+            $message = $_POST['message'];
+            $mailTo = "gaganchhabra24@yahoo.com";
+            $headers = "From: " . $mailFrom;
+            $txt = "You have recieved an email from " . $fullname . ".\n\n" . $message;
+            mail($mailTo, $subject, $txt, $headers);
+            header("Location: contactus.php?mailsend");
+        }
+    }
+        ?>
+
         <h1> Connect with Us! </h1></br>
-        <p>Send E-mail</p>
+        <p>Send E-mail</p><br/>
         <form class="contact-form" action="contactus.php" method="post">
-            <input type="text" name="name" placeholder="Full Name">
-            <input type="text" name="mail" placeholder="Your Email">
-            <input type="text" name="subject" placeholder="Subject">
-            <textarea name="message" placeholder="Message"></textarea>
-            <button type="submit" name="submit">Send Mail</button>
+            <input type="text" name="full name" <?= ($fullname ? 'value="' . $fullname . '"' :  ''); ?> placeholder="Full Name"required><br/>
+            <input type="email" name="mail" <?= ($email ? 'value="' . $email . '"' : ''); ?> placeholder="Your Email" required><br/>
+            <input type="text" names="subject" <?=($subject ? 'value="' . $subject . '"' : ''); ?> placeholder="Subject" required></br/>
+            <textarea name="message" placeholder="Tell us about your experience! Better yet..dont." required><?= ($message ? $message : ''); ?></textarea><br/>
+            <input type="submit" name="submit-btn"></br>
         </form>
-        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
+    </div>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
+<--add checkbox for subscribe for coups
```

Comment:
Whats going on here?

Reply:
forgot to use the proper commenting shortcut. Was for the group to potentially discuss including. Has been removed